src/Bonsai.ML.Visualizers/Bonsai.ML.Visualizers.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Title>Bonsai.ML.Visualizers</Title>
+    <Description>A package for the Bonsai visual programming language.</Description>
+    <Authors />
+    <Copyright>Copyright ©  2024</Copyright>
+    <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
+    <PackageTags>Bonsai Rx Bonsai.ML.Visualizers</PackageTags>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
+    <Features>strict</Features>
+    <Version>0.1.0</Version>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Bonsai.Core" Version="2.8.1" />
+    <PackageReference Include="Bonsai.Design.Visualizers" Version="2.8.0" />
+    <PackageReference Include="OxyPlot.Core" Version="2.1.2" />
+    <PackageReference Include="OxyPlot.WindowsForms" Version="2.1.2" />
+    <PackageReference Include="System.Windows.Forms" Version="4.0.0" />
```

Comment:
`System.Windows.Forms` can probably be removed as well with the frameworks we are targeting at the moment.